### ISSUE TITLE
ter-computed-property-spacing

### ISSUE THREAD

**Issue by jeremejevs** [buzinas/tslint-eslint-rules#326]
The following code is marked as invalid by the "never" mode of this rule:

```ts
const x = obj[
  123
];
```

However, I believe it _is_ valid, as per [ESLint's tests](https://github.com/eslint/eslint/blob/de4dba963963d4fed27dc770e52c2d23fd0542d9/tests/lib/rules/computed-property-spacing.js#L61-L62):

```js
{
  valid: [
    { code: "obj[\nfoo]", options: ["never"] },
    { code: "obj[foo\n]", options: ["never"] }
  ]
}
```
